<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <resources><include location="../../../res/extui/extui.qrc"/></resources><author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>QG_TextOptions</class>
  <widget class="QWidget" name="QG_TextOptions">
    <property name="geometry">
      <rect>
        <x>22</x>
        <y>99</y>
        <width>300</width>
        <height>24</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy>
        <hsizetype>0</hsizetype>
        <vsizetype>0</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="minimumSize">
      <size>
        <width>200</width>
        <height>22</height>
      </size>
    </property>
    <property name="maximumSize">
      <size>
        <width>300</width>
        <height>32767</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Text Options</string>
    </property>
    <property name="toolTip" stdset="0">
      <string/>
    </property>
    <layout class="QHBoxLayout">
      <property name="margin">
        <number>1</number>
      </property>
      <item>
        <widget class="QLabel" name="lText">
          <property name="text">
            <string>Text:</string>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="Q3TextEdit" name="teText">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="minimumSize">
            <size>
              <width>0</width>
              <height>22</height>
            </size>
          </property>
          <property name="maximumSize">
            <size>
              <width>32767</width>
              <height>22</height>
            </size>
          </property>
          <property name="vScrollBarMode">
            <enum>Q3ScrollView::AlwaysOff</enum>
          </property>
          <property name="hScrollBarMode">
            <enum>Q3ScrollView::AlwaysOff</enum>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="lAngle">
          <property name="text">
            <string>Angle:</string>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="leAngle"/>
      </item>
      <item>
        <widget class="Line" name="sep1">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="frameShape">
            <enum>QFrame::VLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <includes>
    <include location="local">rs_line.h</include>
    <include location="local">rs_settings.h</include>
    <include location="local">rs_modification.h</include>
    <include location="local">rs_actiondrawtext.h</include>
  </includes>
  <connections>
    <connection>
      <sender>teText</sender>
      <signal>textChanged()</signal>
      <receiver>QG_TextOptions</receiver>
      <slot>updateText()</slot>
    </connection>
    <connection>
      <sender>leAngle</sender>
      <signal>textChanged(QString)</signal>
      <receiver>QG_TextOptions</receiver>
      <slot>updateAngle()</slot>
    </connection>
  </connections>
</ui>
